<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="OCT-JULIEN" failures="4" tests="8" name="Change Password Test" time="105.179" errors="0" timestamp="2022-07-20T15:39:49 WAT">
  <testcase classname="alphaTests.ChangePasswordTest" name="CheckforThePasswordTextFieldThatIslessThan9characters" time="12.576"/>
  <testcase classname="alphaTests.ChangePasswordTest" name="CheckforThePasswordTextFieldThatHasAmissingNumber" time="2.427"/>
  <testcase classname="alphaTests.ChangePasswordTest" name="CheckforThePasswordTextFieldThatHasAmissingSpecialCharacter" time="2.379"/>
  <testcase classname="alphaTests.ChangePasswordTest" name="CheckforThePasswordTextFieldThatHasAmissingLowercase" time="32.519">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at alphaTests.ChangePasswordTest.ValidateChangePasswordInvalidInputsTrue(ChangePasswordTest.java:193)
at alphaTests.ChangePasswordTest.CheckforThePasswordTextFieldThatHasAmissingLowercase(ChangePasswordTest.java:78)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- CheckforThePasswordTextFieldThatHasAmissingLowercase -->
  <testcase classname="alphaTests.ChangePasswordTest" name="CheckforThePasswordTextFieldThatHasAmissingUppercase" time="32.459">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at alphaTests.ChangePasswordTest.ValidateChangePasswordInvalidInputsTrue(ChangePasswordTest.java:193)
at alphaTests.ChangePasswordTest.CheckforThePasswordTextFieldThatHasAmissingUppercase(ChangePasswordTest.java:87)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- CheckforThePasswordTextFieldThatHasAmissingUppercase -->
  <testcase classname="alphaTests.ChangePasswordTest" name="ValidateChangePasswordUsinginvalidPasswordInOldPasswordField" time="5.361"/>
  <testcase classname="alphaTests.ChangePasswordTest" name="ValidateChangePasswordUsingOldPasswordinNewPasswordField" time="3.537">
    <failure type="java.lang.AssertionError" message="expected [This is an old password, please change it] but found [Invalid credentials]">
      <![CDATA[java.lang.AssertionError: expected [This is an old password, please change it] but found [Invalid credentials]
at alphaTests.ChangePasswordTest.ValidateChangePasswordUsingOldPasswordinNewPasswordField(ChangePasswordTest.java:114)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- ValidateChangePasswordUsingOldPasswordinNewPasswordField -->
  <testcase classname="alphaTests.ChangePasswordTest" name="ValidateChangePasswordUsingValidOldAndNewPasssword" time="3.459">
    <failure type="java.lang.AssertionError" message="expected [Password updated successfully] but found [Invalid credentials]">
      <![CDATA[java.lang.AssertionError: expected [Password updated successfully] but found [Invalid credentials]
at alphaTests.ChangePasswordTest.ValidateChangePasswordUsingValidOldAndNewPasssword(ChangePasswordTest.java:146)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- ValidateChangePasswordUsingValidOldAndNewPasssword -->
</testsuite> <!-- Change Password Test -->
